### PR TITLE
Updating the nav

### PR DESCRIPTION
**Pull request by rlc4**
Apparently if we have too many characters in the nav bar, it wraps and becomes invisible and shoves page titles to be right-justified.

I've gathered the major topics under bigger headings, maintained the Home link, and also added in a link for "Organizers" (the /jobs page).  
Updated the /jobs page to say we're not hiring right now, volunteering is a way to get noticed.

### CODE REVIEW

File: _data/nav.yml [hunk 1]
```diff
@@ -1,10 +1,34 @@
...
   href: "/"
 
-- title: "About"
+
+- title: Attend
+  subcategories:
+    - subtitle: Schedule
+      subhref: /schedule
+    - subtitle: Tickets
+      subhref: /tickets
```
Comment (reedloden) [inline _data/nav.yml:10]:
```suggestion
    - subtitle: "Schedule"
      subhref: "/schedule"
    - subtitle: "Tickets"
      subhref: "/tickets"
    - subtitle: "Venue"
      subhref: "/venue"
```

File: _data/nav.yml [hunk 2]
```diff
@@ -1,10 +1,34 @@
...
+      subhref: "/cfp"
+    - subtitle: "Organizers"
+      subhref: "/jobs"
+
+- title: About
   subcategories:
-    - subtitle: "BSidesSF"
-      subhref: "/about"
+    - subtitle: BSidesSF
+      subhref: /about
```
Comment (reedloden) [inline _data/nav.yml:31]:
```suggestion
    - subtitle: "BSidesSF"
      subhref: "/about"
```

File: _data/nav.yml [hunk 3]
```diff
@@ -1,10 +1,34 @@
...
+    - subtitle: "Friends"
+      subhref: "/friends"
+
+- title: "Volunteers & Presenters"
+  subcategories:
+    - subtitle: "Volunteers"
+      subhref: "/volunteer"
+    - subtitle: "Presenters"
+      subhref: "/cfp"
+    - subtitle: "Organizers"
```
Comment (reedloden) [inline _data/nav.yml:25]: Should we even include this if we're getting rid of this for now?

File: _data/nav.yml [hunk 4]
```diff
@@ -1,10 +1,34 @@
...
+      subhref: /tickets
+
+- title: "Sponsors & Friends"
+  subcategories:
+    - subtitle: "Sponsors"
+      subhref: "/sponsors"
+    - subtitle: "Friends"
+      subhref: "/friends"
+
+- title: "Volunteers & Presenters"
```
Comment (reedloden) [inline _data/nav.yml:19]: I don't think we should be advertising the CFP on the website at all. Presenters can go to /cfp if they want, but it shouldn't be in the nav.

File: _data/nav.yml [hunk 5]
```diff
@@ -1,10 +1,34 @@
...
-- title: "About"
+
+- title: Attend
+  subcategories:
+    - subtitle: Schedule
+      subhref: /schedule
+    - subtitle: Tickets
+      subhref: /tickets
+
+- title: "Sponsors & Friends"
```
Comment (reedloden) [inline _data/nav.yml:12]: Can we keep these separate, if possible?

Comment (rlc4) [reply _data/nav.yml]: Yes, testing this does have enough space to be at the top level

File: _data/nav.yml (hunk 4 above)
Comment (rlc4) [reply _data/nav.yml:19]: Removed.

File: _data/nav.yml (hunk 3 above)
Comment (rlc4) [reply _data/nav.yml:23]: I'm open to either keeping or removing.  I'd like something on the nav for people to watch, and in future apply when we open up for more to join us, without them having to remember /jobs

File: _data/nav.yml [hunk 6]
```diff
@@ -1,7 +1,29 @@
 - title: "Home"
   href: "/"
 
-- title: "About"
+
+- title: Attend
```
Comment (reedloden) [inline _data/nav.yml]:
```suggestion
- title: "Attend"
```

**Review by reedloden — APPROVED** [review]

**Pull request merged**